server/main.go

```diff
@@ -45,7 +43,7 @@ func (app *App) setupRoutes() {
 
 	/* GET endpoint that calls app's ___ function defined in maintenance.go */
 	// r.GET("/maintenance_requests", app.AllMaintenanceRequests())
-
+	app.server.POST("/createmr", app.CreateMaintenanceRequests)
```

Comment:
We should call this endpoint `"/maintenance_requests"` according to proper API naming conventions:

```
GET /maintenance_requests               # Get a collection   
GET /maintenance_requests/:id           # Get a single resource
PUT /maintenance_requests/:id           # Update a single resource
POST /maintenance_requests              # Create a single resource
DELETE /maintenance_requests/:id        # Delete a single resource
```

Reply:
fix: renamed endpoint to /maintenance_requests